src/redux/actions/articleActionCreator.js

```diff
@@ -0,0 +1,42 @@
+import axios from "../../utils/axios";
+import {
+  ARTICLE_FETCHED,
+  FETCHING_ARTICLE,
+  FETCHING_ASIDE_ARTICLES,
+  ARTICLE_ERROR,
+  DELETE_ARTICLE
+} from "../actionTypes";
+
+export const fetchingArticle = () => ({ type: FETCHING_ARTICLE });
+
+export const fetchArticle = slug => async dispatch => {
+  try {
+    dispatch(fetchingArticle);
+
+    const response = await axios.get(`/articles/${slug}`);
+
+    dispatch({ type: ARTICLE_FETCHED, payload: response.data });
+
+    const response2 = await axios.get(`/articles?limit=3`);
```

Comment:
@YvesIraguha Why have you limited to 3?

Reply:
Well, this was done in viewing a single article. However, I limited to 3 articles, because, when they are more than 3 and the main article is very short, styles don't align correctly.